src/Client.cpp

```diff
@@ -59,6 +61,18 @@ std::string Client::getModes() const {
 	return modes.empty() ? "" : "+" + modes;
 }
 
+size_t Client::getJoinedChannelsNbr() const {
+	size_t n = 0;
+
+	for (size_t i = 0; i < Server::getInstance().getChannels().size(); ++i) {
+		if (Server::getInstance().getChannels()[i]->isUserOnChannel(
```

Comment:
Prefere create a local variable to store the array instead
And is not already there a function that returns a list of joined channels ?

Reply:
There wasn't any methods but now there is :)